index.html

```diff
@@ -11,28 +12,49 @@
 
         gtag('config', 'G-KK41RCPR06');
     </script>
-    <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <script src="https://kit.fontawesome.com/31d9495df1.js" crossorigin="anonymous"></script>
-    <title>Portfolio</title>
+    <title>Aku Khan' s Portfolio</title>
     <link rel="stylesheet" href="style.css" />
-    <link rel="stylesheet" href="./assets/all.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
 </head>
 
 <body>
     <div class="main">
         <header class="heading">
-            <a class="logo" href="#"> Aku </a>
+            <a class="logo" href="#"> <h2>Aku </h2> </a>
             <nav class="mobile">
```

Comment:
- [ ] Please make sure the the mobile menu is aligned as per the design. Kindly fix the margins. See images below; 

***The design***
![aakD](https://user-images.githubusercontent.com/65852621/136013131-9a9fc743-b74b-4fa0-a95b-be9c6d34448d.png)


***Your implementation at 400px***
![aak400px](https://user-images.githubusercontent.com/65852621/136013012-17a43864-33c2-4754-ad4d-836dbd111b26.png)

***Your implementation at 320px***
![aak320](https://user-images.githubusercontent.com/65852621/136013592-e17f97d0-6ffc-421b-b6b2-6a339d7dc8f5.png)

Reply:
Thank You. Made Changes.